media/javascript/elastic-search.js

```diff
@@ -0,0 +1,14 @@
+$(document).ready(function () {
+  $('#order-select').change(function () {
+    var selectedOrder = $(this).children('option:selected').val()
+    var params = Qs.parse(window.location.search.replace('?', ''))
+
+    if (selectedOrder && selectedOrder !== 'relevance') {
```

Comment:
`params.sort = (selectedOrder && selectedOrder !== 'relevance') ? selectedOrder : null`

Reply:
Non lo uso mai `delete` per cancellare la chiave di un oggetto, di solito non mi piace, ma se setto `params.sort = null` nell'url vedrò `http://local.docs.italia.it/search/?q=documento&type=file&sort=`

Se va bene effettuo la modifica

Reply:
Mh, vabbé poco male, lascia pure come l'avevi fatto.